backend/support/management/commands/run_2022_23.py

```diff
@@ -16,16 +16,23 @@ class Command(BaseCommand):
     Beware! Deletes any existing rows in SingleAuditChecklist
     """
 
-    def handle(self, *args, **kwargs):
+    def add_arguments(self, parser):
+        parser.add_argument("--year", help="Year(2022 or 2023)", type=str)
```

Comment:
Would it be worth generalizing this command to accept any `year` as an argument to avoid doing a manual update every year?

Reply:
2019 is the current baseline year for cog/over.  The earliest year that we want to test with run2022_23 command will be 2022.  We use UEI, EIN to find dbkey from 2022 data.  The latest year will be 2024.
For 2025, things will change based on the cog/over requirements then.
So, we can allow 2022 through 2024 in the command.